src/main/java/org/example/c02controlflow/generate/Display10Integers.java

```diff
@@ -0,0 +1,17 @@
+package org.example.c02controlflow.generate;
+
+/*
+ Write a program in Java to display the first 10 natural numbers.
```

Comment:
Modify to: Write a program in Java to display the first n natural numbers.
Fix the implementation to display n natural numbers instead of number + 10.

Reply:
modificat